### PR TITLE
[14.0][MIG] shopinvader_sale_packaging

### PR DESCRIPTION
**Pull request by hparfr**
Depends of : 

- [x] sale_by_packaging https://github.com/OCA/sale-workflow/pull/1527 (replaces 1398)
- [x] sale_order_line_packaging_qty https://github.com/OCA/sale-workflow/pull/1396
- [x] product_packaging_type https://github.com/OCA/product-attribute/pull/760/
- [x] stock_packaging_calculator https://github.com/OCA/stock-logistics-warehouse/pull/1050

### CODE REVIEW

**Review by PierrickBrun — APPROVED** [review]
Do not merge until test-requirements PRs are merged

File: shopinvader_sale_packaging/services/abstract_sale.py
Comment (acsonefho) [inline shopinvader_sale_packaging/services/abstract_sale.py]: **Tips:** you can use the `one=True` parameter
https://github.com/OCA/server-tools/blob/14.0/base_jsonify/models/models.py#L93

```suggestion
                    "packaging": line.product_packaging.jsonify(["id", "name"], one=True),
```

**Review by acsonefho — APPROVED** [review]
Code LGTM

**Review by simahawk — CHANGES_REQUESTED** [review]
Quick update: rebase required, OCA PRs still pending, fix/imp to cherry pick from v13

**Comment by sebastienbeau** [comment]
@simahawk I have cherry-pick the IMP here : https://github.com/shopinvader/odoo-shopinvader/pull/1028
Do you see other FIX/PR to pick ?

**Comment by simahawk** [comment]
> @simahawk I have cherry-pick the IMP here : #1028
> Do you see other FIX/PR to pick ?

just check the log on v13 https://github.com/shopinvader/odoo-shopinvader/commits/13.0/shopinvader_sale_packaging

there are several commits missing before that one:

https://github.com/shopinvader/odoo-shopinvader/commit/3928aee96e9e48c344249679f08cb8f37740ca78
https://github.com/shopinvader/odoo-shopinvader/commit/479aae0292fae500b1f4ca2e142af495657bf8fe
https://github.com/shopinvader/odoo-shopinvader/commit/d0dc09e6a32a811e54573be94206b3ae2fdb3fc1
https://github.com/shopinvader/odoo-shopinvader/commit/c083aa62721d5079c5e7627757699c5e8a2d9ff7
https://github.com/shopinvader/odoo-shopinvader/commit/304f9baee18c017b0938c31e22c116d5aa759d21
https://github.com/shopinvader/odoo-shopinvader/commit/b586d64ea00e0fc1899fe4849a0c852a73b84845

**Comment by simahawk** [comment]
@hparfr @PierrickBrun hello :)
Is anyone going further with this?

**Comment by hparfr** [comment]
@sebastienbeau will you continue the cherry-pick :cherries:  ?

**Comment by simahawk** [comment]
@sebastienbeau ping :)

**Comment by kevinkhao** [comment]
I think we can continue over there https://github.com/shopinvader/odoo-shopinvader/pull/1131 because it seems cleaner to do a new one

**Pull request closed without merging**
